src/gui_common/MainMenu.cs

```diff
@@ -29,9 +29,11 @@ public class MainMenu : Node
     [Export]
     public NodePath FreebuildButtonPath;
 
+    [Export]
+    public NodePath GLES2PopupPath;
+
     public Array MenuArray;
     public TextureRect Background;
-
```

Comment:
Please put this blank line back in, I think it's nice to separate logically different fields.